install/kubernetes/cilium/values.schema.json

```diff
@@ -5324,4 +5330,4 @@
   },
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object"
-}
\ No newline at end of file
+}
```

Comment:
This newline change seems off. If you use the Makefile to generate the schema (`make -C install/kubernetes`), I believe this should be fixed

Reply:
My bad that's probably my editor. I also had to change `values.yaml.tmpl` before running the Makefile(s).